IPyth.sol

```diff
@@ -50,8 +52,16 @@ interface IPyth {
     /// @return publishTime - the UNIX timestamp of when this price was computed.
     function getPrevPriceUnsafe(bytes32 id) external view returns (PythStructs.Price memory price, uint64 publishTime);
 
-    /// @notice Update price feeds with given update messages if they are more recent than the current stored prices.
+    /// @notice Update price feeds with given update messages.
+    /// You should transfer some fee in Wei when calling this method; You can get its amount by calling `getMinUpdateFee`.
```

Comment:
This method requires the caller to pay a fee in wei; the required fee can be computed by calling `getMinUpdateFee` with the length of the `updateData` array. 

In general, I suggest trying to avoid "you" in code comments.

Reply:
also nitpick: you don't capitalize the first word after a semicolon. It's the same sentence.

Reply:
thanks!